### PR TITLE
Linked list to array  - U19CS076 

### PR DESCRIPTION
**Pull request by krithikha2001**
ADM NO-U19CS076

### CODE REVIEW

File: stack.c [hunk 1]
```diff
@@ -1,59 +1,65 @@
...
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "stackconfig.h"
+#include "stack.h"
+
+//#define MAX 10
+void push(Stack* stack, stackType data) {
+   // Node* newNode = malloc(sizeof(Node));
+    if(stack->top == _MAX) {
```
Comment (murtraja) [inline stack.c:10]: _MAX -1?

**Comment by murtraja** [comment]
Great! Looks good to me.

File: stack.c (hunk 1 above)
Comment (krithikha2001) [reply stack.c:10]: Sorry sir, didn't notice- by mistake. I have updated the code locally. Shall I pr again?

Comment (murtraja) [reply stack.c:10]: up to you

**Pull request closed without merging**